modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -2717,14 +2742,23 @@ bool InputMediaStream_FFMPEG::read(unsigned char** data, int* size, int* endOfFi
             continue;
         }
 
+        result = true;
         break;
     }
 
-    *data = pkt_.data;
-    *size = pkt_.size;
-    *endOfFile = false;
+#if USE_AV_INTERRUPT_CALLBACK
```

Comment:
Btw, should these values be read from current environment? Because then if you'd want to disable current behavior you could set an infinite value.